server/plugins/client/bundle.js

```diff
@@ -31,6 +31,14 @@ function bundleClient (hoodieClientPath, bundleTargetPath, config, callback) {
   var plugins = [
     path.resolve('hoodie/client')
   ].filter(checkModule)
+
+  var thirdPartyPlugins = require('hoodie.plugins')
+  $.each(thirdPartyPlugins, function(){
+    concat(
+    path.resolve('hoodie/client').filter(checkModule)
+    )
+  })
```

Comment:
`$.each` is a jQuery function, and we don’t use jQuery in the Hoodie server code. What you can do instead is this:

```js
thirdPartyPlugins.foreach(function (pluginName) {
  var pluginPath = pluginName + '/hoodie/client'
  if (checkModule(pluginPath)) {
    plugins.push(require.resolve(pluginPath))
  }
})
```

[require.resolve](https://nodejs.org/api/globals.html#globals_require_resolve) returns the full path to the file we want to bundle into the client.